colors/PaperColor.vim

```diff
@@ -1130,6 +1130,7 @@ fun! s:apply_syntax_highlightings()
     " Switching between dark & light variant through `set background`
     if s:is_dark " DARK VARIANT
       set background=dark
+      exec 'hi EndOfBuffer' . s:fg_cursor_fg  . s:ft_none
```

Comment:
I wonder what is that for?

Reply:
@pickfire  it's set the tilde  fg color same as background，so you  won't see it both after the vim is started。 like this gif
![go](https://user-images.githubusercontent.com/41671631/54993121-14c02380-4ffc-11e9-99d8-1700cc410c76.gif)

Reply:
@taigacute Oh, so that is the other part of the patch to hide tilde in dark mode.

I think it should be separated as another patch, I thought both of it is related to startify and it might confuse others.

In my humble opinion, I don't quite like hiding the tilde. I personally uses the light theme more but I prefer it to stay there to know if it is part of the file.

Reply:
@pickfire  you are right It should be a separate patch, the tilde is ugly, I think, but everyone's ideas are different and it's hard to agree. This is my personal suggestion. Hiding the tilde in dark mode is better looking at me

Reply:
I think we can leave that for @NLKNguyen to decide but he can make choices to accept the patch easier if the patch are split into two. @taigacute Thanks a lot for the sending in the patches by the way. :+1:

Reply:
ok  I will resubmit pr。

Reply:
Thanks, I'm looking forward to the new patch. It does look better without the tildes on empty lines. 👍

Reply:
thanks  i will submit pr